R/bootstrap.nlsfit.R

```diff
@@ -104,10 +104,19 @@ parametric.bootstrap.cov <- function (boot.R, x, cov, seed) {
 #' fit.result <- parametric.nlsfit(fn, c(1, 1), boot.R, value, dvalue, x, dx)
 #' summary(fit.result)
 parametric.nlsfit <- function (fn, par.guess, boot.R, y, dy, x, dx,
+                               lower, upper,
                                ..., bootstrap=TRUE, na.rm = FALSE) {
   stopifnot(length(x) == length(y))
   stopifnot(missing(dx) || length(dx) == length(x))
   stopifnot(missing(dy) || length(dy) == length(y))
+  if( missing(lower) ){
+    lower <- rep(x = -Inf, times = length(par.guess))
+  }
```

Comment:
Why don't you just give this as a default value in the function signature?

Reply:
why easy and clean when hard is an option? I was just not thinking about the fact that  the default arguments can be rather elaborate in R...